bigchaindb/pipelines/block.py

```diff
@@ -166,7 +166,7 @@ def create_pipeline():
     """Create and return the pipeline of operations to be distributed
     on different processes."""
 
-    block = Block()
+    block = BlockPipeline()
 
     block_pipeline = Pipeline([
```

Comment:
@sohkai can you please create a ticket for this as we have not even handled your comment